crates/transaction-pool/src/maintain.rs

```diff
@@ -535,6 +576,91 @@ fn changed_accounts_iter(
         .map(|(address, acc)| ChangedAccount { address, nonce: acc.nonce, balance: acc.balance })
 }
 
+async fn apply_local_txs_backup<P>(pool: P, transactions_path: Option<PathBuf>)
```

Comment:
these cli helper functions should return `Result` instead. if the error needs to be ignored, you can do it w/ `if let Err(_) = { /* noop */ }`

Reply:
fixed